src/backend/launcher.ts

```diff
@@ -761,9 +755,15 @@ export async function verifyWinePrefix(
   const haveToWait = !existsSync(systemRegPath)
 
   const command = runWineCommand({
-    commandParts: ['wineboot', '--init'],
+    commandParts:
+      wineVersion.type === 'proton' &&
+      GlobalConfig.get().getSettings().experimentalFeatures?.umuSupport !==
+        false
+        ? ['createprefix']
+        : ['wineboot', '--init'],
     wait: haveToWait,
     gameSettings: settings,
+    protonVerb: 'run',
```

Comment:
By the way, if you omit the verb, umu will default to using `waitforprefixandrun` verb which will block users from running more than 1 game in the same prefix. However, if heroic wants to allow users to run more than 1 game, then you could default to `runinprefix`.

Reply:
we use `waitforexitandrun` only for game executables to ensure we run in proper env. There is no reason for users to use same prefix for more than one game.
`waitforexitandrun` acts as a safeguard to avoid situations where wineserver instance runs with fsync and the game needs that disabled.